webdriver-spec.html

```diff
@@ -6327,7 +6327,7 @@
  and a <dfn>source type</dfn> which determines
  the kind of input the device can provide.
  As with real devices, virtual devices are stateful;
- this state is recorded in an <a data-lt="input state">input device state</a> object
+ this state is recorded in an <a>input state table</a>
```

Comment:
The state of the virtual devices is held by the input source, and these are what is held in the input state table. 

This might be the best place to define what "device state" is: a general term for the persistent state held by each input source.